### PR TITLE
Update vignette, cubic post fx

### PR DESCRIPTION
**Pull request by sigvef**

### CODE REVIEW

**Comment by cristeahub** [comment]
This sort of happens when I checkout this branch and the `fxfx` branch in nin.
untiteled

File: src/CubicLensDistortionLayer.js [hunk 1]
```diff
@@ -5,7 +5,10 @@ function CubicLensDistortionLayer() {
   this.shaderPass = new THREE.ShaderPass(SHADERS.cubicLensDistortion);
 }
 
-CubicLensDistortionLayer.prototype.update = function() {
+CubicLensDistortionLayer.prototype.update = function(frame, relativeFrame) {
+  a = this.shaderPass;
```
Comment (stianjensen) [inline src/CubicLensDistortionLayer.js]: debug?

**Comment by sigvef** [comment]
ping

**Comment by peterhgombos** [comment]
Rebase please.

**Comment by sigvef** [comment]
ping

**Comment by peterhgombos** [comment]
Playback stops when moving from intro to desertlayer

**Pull request merged**